honeycomb-kernels/src/grisubal/model.rs

```diff
@@ -237,28 +237,37 @@ pub struct MapEdge<T: CoordsFloat> {
     pub end: DartIdentifier,
 }
 
-#[derive(Clone, Copy, Debug)]
-pub struct IsBoundary(bool);
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum Boundary {
+    Left,
+    Right,
+    None,
+}
```

Comment:
Can you comment on what it means?

Reply:
I've added comments & a figure in 29aac32